connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectStreamHandler.scala

```diff
@@ -52,19 +52,15 @@ class SparkConnectStreamHandler(responseObserver: StreamObserver[ExecutePlanResp
     session.withActive {
 
       // Add debug information to the query execution so that the jobs are traceable.
-      try {
```

Comment:
I removed this try-catch for test purpose, will add it back to be more conservative

Reply:
I think it's fine to remove it (since you added it to address this specific case before?). I don't mind removing it

Reply:
yes, I added it for that PyTorch test case, in which the size of UDF is 47mb and cause OOM

but I am not very sure whether there are some unknown edge cases that can also cause failure, so I personally prefer adding the try-catch back before merge it.